spec/cadmium/tokenizer/aggressive_tokenizer_spec.cr

```diff
@@ -27,6 +27,10 @@ describe Cadmium::AggressiveTokenizer do
     it "should swollow duplicate punctuation" do
       subject.tokenize("i shal... pause").should eq(["i", "shal", "pause"])
     end
+
+    it "should return empty string when fed with empty string" do
+      subject.tokenize("").should eq([""])
```

Comment:
This shouldn't be correct. `tokenize`, when ran on an empty string, should return an empty `Array(String)`.

Reply:
You mean the spec description is wrong and should be : 
`"should return an empty Array(String)when fed with empty string"`

Or the value returned is wrong ?

Reply:
The expectation is wrong. Rather than the return value being `[""]` it should be `[] of String`. I detailed a better fix for #3 below.